AnnoyancesFilter/Other/sections/self-promo.txt

```diff
@@ -1879,6 +1879,8 @@ crisisgroup.org##.c-cta-block
 thenation.com##.indent.current-issue
 thenation.com##.inline-cta-1
 examine.com##.optin-wrapper
+currentaffairs.org##.wp-block-image:has(> a[href$="/subscribe"])
```

Comment:
This rule should be Annoyances - subscriptions_specific.txt
`currentaffairs.org#?#.wp-block-image:has(> a[href$="/subscribe"])`

Reply:
even though it's not a popup?

Reply:
Yes. The main point is that this element is a subscription.
I'd use the similar rule, but with full link. I don't see any differences in the code.
```
currentaffairs.org#?#.wp-block-image:has(> a[href="http://currentaffairs.org/subscribe"])
```

Reply:
> I'd use the similar rule, but with full link. I don't see any differences in the code.

https://github.com/AdguardTeam/AdguardFilters/pull/128530#discussion_r960757684